test/e2e/framework/providers/azure/azure.go

```diff
@@ -1,6 +1,3 @@
-//go:build !providerless
```

Comment:
Not related to this PR, but I am surprised that I see this file. I thought we removed azure support in v1.30.